### PR TITLE
#98 close the time selector when the user switches between inputs with tab

### PR DESCRIPTION
**Pull request by rezk2ll**
demo: https://streamable.com/sm7cv1

updated demo: https://streamable.com/43f4u5

Continues to resolve https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/98.

### CODE REVIEW

File: src/esn.calendar.libs/app/components/time-select/time-select.controller.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ function calTimeSelectController($scope, $element, calMoment) {
   }
 
   function onInputKeydown(event, mdMenu) {
+    if (event.key === 'Tab') return mdMenu.close();
```
Comment (tuanlc) [inline src/esn.calendar.libs/app/components/time-select/time-select.controller.js:32]: People will break you awesome improvement if there is no test for it ;)

**Review by tuanlc — CHANGES_REQUESTED** [review]

**Comment by huy-ta** [comment]
Well I have just noticed that when you tab between inputs, the time picker should be showing when it's either one of the time inputs, but right now it doesn't.

File: src/esn.calendar.libs/app/components/time-select/time-select.controller.js (hunk 1 above)
Comment (rezk2ll) [reply src/esn.calendar.libs/app/components/time-select/time-select.controller.js:32]: done, thanks

**Comment by rezk2ll** [comment]
> Well I have just noticed that when you tab between inputs, the time picker should be showing when it's either one of the time inputs, but right now it doesn't.

it does now.

**Review by huy-ta — APPROVED** [review]

**Review by asmaNode — APPROVED** [review]

**Pull request merged**